src/components/js-tracer/Settings.js

```diff
@@ -41,7 +42,9 @@ class JsTracerSettingsImpl extends PureComponent<Props> {
                 onChange={this._onCheckboxChange}
                 checked={showJsTracerSummary}
               />
-              {' Show only self time'}
```

Comment:
We might want to keep this `{' '}` at the start of the string?

*Update*: I'm not sure we need it after I saw the other components. Maybe just ignore this comment, I guess we'll have to check this if/when we reenable this component.

Reply:
Yeah, when I check this locally, it wasn't actually needed and it didn't make a difference.